### PR TITLE
osdc: Reverse the probing vector in better way.

### PR DESCRIPTION
**Pull request by songbaisen**
osdc: Reverse the probing vector in better way.

Signed-off-by: song baisen song.baisen@zte.com.cn

### CODE REVIEW

File: src/osdc/Filer.cc [hunk 1]
```diff
@@ -209,12 +209,7 @@ bool Filer::_probed(Probe *probe, const object_t& oid, uint64_t size,
 
   if (!probe->fwd) {
     // reverse
-    vector<ObjectExtent> r;
-    for (vector<ObjectExtent>::reverse_iterator p = probe->probing.rbegin();
-	 p != probe->probing.rend();
-	 ++p)
-      r.push_back(*p);
-    probe->probing.swap(r);
+    std::reverse(probe->probing.begin(),probe->probing.end());
```
Comment (tchaikov) [inline src/osdc/Filer.cc]: could you add a space after the comma, and remove the comment? as the code is self explained =)

**Comment by songbaisen** [comment]
@tchaikov  Good morning!Thank you for your review.I have change it.

**Comment by tchaikov** [comment]
@songbaisen nit, better off including `<algorithm>` explicitly in this .cc file.

otherwise lgtm.

**Comment by songbaisen** [comment]
@tchaikov  Morning sir.Add it!

**Comment by tchaikov** [comment]
lgtm

**Pull request merged**